cypress/integration/login.js

```diff
@@ -20,5 +20,12 @@ describe('Login functionality', function() {
         cy.get('.tile.form').should('be.visible')
         cy.get('.user-info__title').contains('TestCon User 6')
         cy.get('.main-nav').find('li').should('have.length', 2)
+
+        let d = new Date();
+        let current_day = d.getDate();
+
+        cy.get('.calendar--selected').should('contain', current_day)
```

Comment:
Nice, almost there! '.calendar--selected' is not the best selector for today's date, as this one will change the date value when you select another day in the calendar

Reply:
Oh yeah a very good point! Didn't check that!